tests/caasp/stack_bootstrap.pm

```diff
@@ -130,8 +130,17 @@ sub kubectl_config {
 }
 
 sub run {
-    assert_screen 'velum-bootstrap-page', 90;
     barrier_wait {name => "WORKERS_INSTALLED", check_dead_job => 1};
+    if (check_screen('velum-sorry', 300)) {
```

Comment:
check_screen has one negative side-effect - if it does not match, you have to wait the timeout (300s).
You expect bug#1074836 and if it's not there you delay for 5 minutes (all QA tests). You can wait for multiple needles this way:
```
assert_screen [qw(velum-bootstrap-page velum-sorry)], 90;
if (match_has_tag 'velum-sorry') {
 ...workaround...
}
```

Reply:
Thanks for the tip. I didn't know that.